pkg/exporter/sumologicexporter/exporter.go

```diff
@@ -296,8 +301,7 @@ func (se *sumologicexporter) pushLogsData(ctx context.Context, ld pdata.Logs) er
 				currentMetadata = sdr.filter.filterIn(attributes)
 
 				if se.config.TranslateAttributes {
-					translateAttributes(attributes)
-					translateAttributes(currentMetadata.orig)
+					currentMetadata.orig = translateAttributes(currentMetadata.orig)
```

Comment:
maybe test are failing due to missing `attributes = translateAttributes(attributes)` :thinking: